__tests__/app.test.js

```diff
@@ -332,4 +332,30 @@ describe.only("DELETE /api/comments/:comment_id", () => {
       expect(body.msg).toBe("404: not found")
     })
   });
-})
\ No newline at end of file
+});
+
+describe("GET /api/users", () => {
+  test("200: Responds with all users", () => {
+    return request(app)
+    .get("/api/users")
+    .expect(200)
+    .then(({body})=>{
+      expect(body.users.length).toBe(4);
+
+      body.users.forEach((user) => {
+        expect(typeof user.username).toBe("string")
+        expect(typeof user.name).toBe("string")
+        expect(typeof user.avatar_url).toBe("string")
+      })
+    })
+  });
+
+  test("404: Error when endpoint not found" ,() => {
+    return request(app)
+    .get("/api/not-users")
```

Comment:
This test isn't specific to this endpoint. If someone puts an incorrect path after /api/ it is a general server error, so should be within its own describe block for general 404 incorrect paths 👍